### PR TITLE
add clarifying access url note

### PR DESCRIPTION
**Pull request by mirnawong1**
adding clearer access url info so users have clear guidance on what they need to add for their own region

### CODE REVIEW

File: website/docs/docs/deploy/dashboard-status-tiles.md [hunk 1]
```diff
@@ -31,42 +31,72 @@ In order to set up your dashboard status tile, here is what you need:
...
-https://metadata.cloud.getdbt.com/exposure-tile?name=<exposure_name>&jobId=<job_id>&token=<metadata_only_token>
+https://metadata.YOUR_ACCESS_URL/exposure-tile?name=<exposure_name>&jobId=<job_id>&token=<metadata_only_token>
+```
+
+:::tip Replace `YOUR_ACCESS_URL` with your region and plan's Access URL
+
+dbt Cloud is hosted in multiple regions in the world and each region has a different access URL. Users can embed an iFrame and replace `YOUR_ACCESS_URL` with the appropriate [Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan. For example, if your account is hosted in an APAC region, you would use the following code:
+
+```
+https://metadata.au.dbt.co/exposure-tile?name=<exposure_name>&jobId=<job_id>&token=<metadata_only_token>
```
Comment (mirnawong1) [inline website/docs/docs/deploy/dashboard-status-tiles.md]:
```suggestion
https://metadata.au.dbt.com/exposure-tile?name=<exposure_name>&jobId=<job_id>&token=<metadata_only_token>
```

File: website/docs/docs/deploy/dashboard-status-tiles.md [hunk 2]
```diff
@@ -31,42 +31,72 @@ In order to set up your dashboard status tile, here is what you need:
...
 Sigma does not require you to embed an iFrame. Add a new embedded UI element in your Sigma Workbook in the following format:
 
 ```
-https://metadata.cloud.getdbt.com/exposure-tile?name=<exposure_name>&jobId=<job_id>&token=<metadata_only_token>
+https://metadata.YOUR_ACCESS_URL/exposure-tile?name=<exposure_name>&jobId=<job_id>&token=<metadata_only_token>
+```
+
+:::tip Replace `YOUR_ACCESS_URL` with your region and plan's Access URL
+
+dbt Cloud is hosted in multiple regions in the world and each region has a different access URL. Users can embed an iFrame and replace `YOUR_ACCESS_URL` with the appropriate [Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan. For example, if your account is hosted in an APAC region, you would use the following code:
```
Comment (mirnawong1) [inline website/docs/docs/deploy/dashboard-status-tiles.md]:
```suggestion
dbt Cloud is hosted in multiple regions in the world and each region has a different access URL. Users can embed an iFrame and replace `YOUR_ACCESS_URL` with the appropriate [Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan. For example, if your account is hosted in the APAC region, you would use the following code:
```

File: website/docs/docs/deploy/dashboard-status-tiles.md [hunk 3]
```diff
@@ -31,42 +31,72 @@ In order to set up your dashboard status tile, here is what you need:
 You can insert these three fields into the following iFrame, and then embed it **anywhere that you can embed an iFrame**:
 
 ```
-<iframe src='https://metadata.cloud.getdbt.com/exposure-tile?name=<exposure_name>&jobId=<job_id>&token=<metadata_only_token>' title='Exposure Status Tile'></iframe>
+<iframe src='https://metadata.YOUR_ACCESS_URL/exposure-tile?name=<exposure_name>&jobId=<job_id>&token=<metadata_only_token>' title='Exposure Status Tile'></iframe>
 ```
 
+:::tip Replace `YOUR_ACCESS_URL` with your region and plan's Access URL
+
+dbt Cloud is hosted in multiple regions in the world and each region has a different access URL. Users can embed an iFrame and replace `YOUR_ACCESS_URL` with the appropriate [Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan. For example, if your account is hosted in an EMEA region, you would use the following iFrame code:
```
Comment (mirnawong1) [inline website/docs/docs/deploy/dashboard-status-tiles.md]:
```suggestion
dbt Cloud is hosted in multiple regions in the world and each region has a different access URL. Users can embed an iFrame and replace `YOUR_ACCESS_URL` with the appropriate [Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan. For example, if your account is hosted in the EMEA region, you would use the following iFrame code:
```

File: website/docs/docs/deploy/dashboard-status-tiles.md [hunk 4]
```diff
@@ -31,42 +31,72 @@ In order to set up your dashboard status tile, here is what you need:
...
 ### Tableau
 Tableau does not require you to embed an iFrame. You only need to use a Web Page object on your Tableau Dashboard and a URL in the following format:
 
+```
+https://metadata.YOUR_ACCESS_URL/exposure-tile?name=<exposure_name>&jobId=<job_id>&token=<metadata_only_token>
+```
+
+:::tip Replace `YOUR_ACCESS_URL` with your region and plan's Access URL
+
+dbt Cloud is hosted in multiple regions in the world and each region has a different access URL. Users can embed an iFrame and replace `YOUR_ACCESS_URL` with the appropriate [Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan. For example, if your account is hosted in a North American region, you would use the following code:
```
Comment (mirnawong1) [inline website/docs/docs/deploy/dashboard-status-tiles.md]:
```suggestion
dbt Cloud is hosted in multiple regions in the world and each region has a different access URL. Users can embed an iFrame and replace `YOUR_ACCESS_URL` with the appropriate [Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan. For example, if your account is hosted in the North American region, you would use the following code:
```

**Comment by mirnawong1** [comment]
hey @Victoriapm , opened up this pr to address your excellent feedback! let me know what you think - deploy preview is [here](https://deploy-preview-3713--docs-getdbt-com.netlify.app/docs/deploy/dashboard-status-tiles)

**Review by Victoriapm — APPROVED** [review]
Looks great! Thanks for the quick improvement as usual 🚀

**Pull request merged**